src/modules/facility/dto/base-facility-request.dto.ts

```diff
@@ -0,0 +1,172 @@
+import { EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { ValidatedCurrencyApiProperty } from '@ukef/decorators/validated-currency-api-property-decorator';
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedDealIdentifierApiProperty } from '@ukef/decorators/validated-deal-identifier-api-property.decorator';
+import { ValidatedFacilityIdentifierApiProperty } from '@ukef/decorators/validated-facility-identifier-api-property';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedPartyIdentifierApiProperty } from '@ukef/decorators/validated-party-identifier-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateOnlyString } from '@ukef/helpers';
+
+export class BaseFacilityRequestItem {
+  @ValidatedDealIdentifierApiProperty({
+    description: 'The identifier of the deal for the facility.',
+  })
+  readonly dealIdentifier: string;
+
+  @ValidatedFacilityIdentifierApiProperty({
+    description: 'The identifier of the facility.',
+  })
+  readonly facilityIdentifier: string;
+
+  @ValidatedPartyIdentifierApiProperty({
+    description:
+      'Look up the Obligors ACBS Customer record using Party URN on the Deal = Alternate Customer Id (J$MSZI). Use ACBS Customer Id (J$MRUI) Note there may be multiple customers in ACBS with the same Party URN. Use the first record found.',
+  })
+  readonly dealBorrowerIdentifier: string;
+
+  @ValidatedStringApiProperty({
+    description: `The facility type, e.g. '250' for BOND.`,
```

Comment:
Can we list all the possible values please? `The product type identifier: 250 for BSS, 260 for EWCS, 280 for GEF-Cash, 281 for GEF-Contingent. It is called the product type code in ACBS.`

Reply:
Leaving as is for now but raising a ticket for this.